scenarios/6_camp/monarch_mod37/cb05_mechanism.noR142R143.json

```diff
@@ -0,0 +1,3112 @@
+{ "camp-data" : [
```

Comment:
Is this the mechanism we used in the paper? Or was it `cb05_mechanism.json`? I would suggest only including one and maybe naming it `cb05_mechanism.json`